includes/wporg-meeting-posttype.php

```diff
@@ -72,11 +72,18 @@ public function meeting_archive_page_query( $query ) {
 	}
 
 	public function meeting_set_next_meeting( $posts, $query ) {
+		if ( !is_array( $posts ) || empty( $posts ) || 'meeting' !== $posts[0]->post_type ) {
+			return $posts;
+		}
```

Comment:
Might be clearer to check for `in_array( (array) $query->post_type, [ 'meeting' ], true )`.

Reply:
Wouldn't `post_type` only be set on meeting archive pages? We want to run this on any requests that contain a meeting, since we don't use the archive pages anymore. Maybe `in_array( 'meeting', wp_list_pluck( $posts, 'post_type' ), true )` would be more clear.

Reply:
You're right, I fixed the order of the needle/haystack arguments.